frontend/src/pages/destructionlist/hooks.ts

```diff
@@ -24,12 +24,16 @@ import { ExpandZaak, Zaak } from "../../types";
 /** The template used to format urls to an external application providing zaak details. */
 const REACT_APP_ZAAK_URL_TEMPLATE = process.env.REACT_APP_ZAAK_URL_TEMPLATE;
 
+interface PaginatedZakenResultsWithAction extends PaginatedZaken {
```

Comment:
What is `PaginatedZakenResultsWithAction`? And does it reflect a backend response?